src/sbf.cpp

```diff
@@ -501,7 +503,8 @@ int GPSDriverSBF::payloadRxDone()
 
 		// Check boundaries and invalidate position
 		// We're not just checking for the do-not-use value (-2*10^10) but for any value beyond the specified max values
-		if (fabs(_buf.payload_pvt_geodetic.latitude) > M_PI_2 || fabs(_buf.payload_pvt_geodetic.longitude) > M_PI ||
+		if (fabs(_buf.payload_pvt_geodetic.latitude) > (double) M_PI_2_F ||
+		    fabs(_buf.payload_pvt_geodetic.longitude) > M_PI_PRECISE ||
```

Comment:
Probably better to avoid px4 defines.h specifics like M_PI_PRECISE, this repo is used by other projects like QGC.

Reply:
Okay, went with M_PI_F then even though that is also defined in px4 defines.h